.github/workflows/ci.yml

```
@@ -21,5 +21,4 @@ jobs:
     - run: uv sync
 
     - uses: astral-sh/ruff-action@v3
-      with:
-        src: './src'
\ No newline at end of file
+    - run: ruff format --check --diff
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)